### ISSUE TITLE
TTL texture issue, duplicate TTL scripts, and warning about mod conflict

### ISSUE THREAD

**Issue by aubergine10** [CitiesSkylinesMods/TMPE#714]
> **AKscene**:
> Hiya. When I load my game, (sometimes) I get a box pop up saying TMPE has **"detected an incompaitibility with another MOD**. You can continue playing but....." So, following the instructiions on your links I went to the TMPE.log and found where it said there was an error. But, it didn't indicate what MOD it was conflicting with. Is there a way to find the conflicting MOD. I am an average player without huge computer knowledge, so hopefully I won't need a degree in Applied Particle Physics to sort this.  
> Also I have noticed since the update, every timed traffic light I have (and there are MANY) has **duplicated itself in the setup timing**. This is not an issue but...strange. Thanks for your advice and assistance.

[TMPE(3).log](https://github.com/CitiesSkylinesMods/TMPE/files/4221997/TMPE.3.log)
[output_log(13).txt](https://github.com/CitiesSkylinesMods/TMPE/files/4222000/output_log.13.txt)

Error in TMPE log (2 of these):

```diff
- Error 310.2440854:   at TrafficManager.UI.Textures.TextureResources.ReadToEnd (System.IO.Stream stream) [0x00000] in <filename unknown>:0 
  at TrafficManager.UI.Textures.TextureResources.LoadDllResource (System.String resourceName, Int32 width, Int32 height) [0x00000] in <filename unknown>:0 
   at CSUtil.Commons.Log.LogToFile(System.String log, LogLevel level)
   at CSUtil.Commons.Log.Error(System.String s)
   at TrafficManager.UI.Textures.TextureResources.LoadDllResource(System.String resourceName, Int32 width, Int32 height)
   at TrafficManager.UI.Textures.TrafficLightTextures..cctor()
   at TrafficManager.UI.SubTools.TimedTrafficLightsTool.ShowGUIOverlay(ToolMode toolMode, Boolean viewOnly)
   at TrafficManager.UI.SubTools.TimedTrafficLightsTool.ShowGUIOverlay(ToolMode toolMode, Boolean viewOnly)
   at TrafficManager.UI.TrafficManagerTool.OnToolGUI(UnityEngine.Event e)
   at ToolBase.OnGUI()
```

Seen similar error in #704

Also getting this which I've never seen before:

```diff
- Error 1,829.5880138: TimedTrafficLightsTool._guiTimedControlPanel: System.ArgumentException: GUILayout: Mismatched LayoutGroup.Repaint
  at UnityEngine.GUILayoutUtility.BeginLayoutGroup (UnityEngine.GUIStyle style, UnityEngine.GUILayoutOption[] options, System.Type layoutType) [0x00000] in <filename unknown>:0 
  at UnityEngine.GUILayout.BeginHorizontal (UnityEngine.GUIContent content, UnityEngine.GUIStyle style, UnityEngine.GUILayoutOption[] options) [0x00000] in <filename unknown>:0 
  at UnityEngine.GUILayout.BeginHorizontal (UnityEngine.GUILayoutOption[] options) [0x00000] in <filename unknown>:0 
  at TrafficManager.UI.SubTools.TimedTrafficLightsTool.GuiTimedControlPanel (Int32 num) [0x00000] in <filename unknown>:0 
   at CSUtil.Commons.Log.LogToFile(System.String log, LogLevel level)
   at CSUtil.Commons.Log.Error(System.String s)
   at TrafficManager.UI.SubTools.TimedTrafficLightsTool.GuiTimedControlPanel(Int32 num)
   at UnityEngine.GUILayout+LayoutedWindow.DoWindow(Int32 windowID)
   at UnityEngine.GUI.CallWindowDelegate(UnityEngine.WindowFunction func, Int32 id, Int32 instanceID, UnityEngine.GUISkin _skin, Int32 forceRect, Single width, Single height, UnityEngine.GUIStyle style)
```

Requested save game for further investigation.

**Comment by aubergine10** [comment]
Potential mod conflicts (just guessing at this stage):

* 1270675750   BigCity
* 932192868    Road Colors Changer ++
* 1586027591   Tiny Segments 3.0
* 563229150    Advanced Toolbar
* 1922400472   Enhanced District Services 1.0.18
* 1456061226   Better Cemetery AI
* 1625704117   Roundabout Builder

**Comment by AnthonyB1005** [comment]
Hi again  aubergine18. Here is the savegame. Hope it helps.

https://steamcommunity.com/sharedfiles/filedetails/?id=2001416919

**Comment by aubergine10** [comment]
This should hopefully be fixed in the 11.1.1 release.
